app/assets/stylesheets/survey_index.scss

```diff
@@ -0,0 +1,32 @@
+@import "colors";
+@import "foundation_and_overrides";
+
+
+
+.button.success.radius.small {
+  margin-left: 170px;
+  margin-right: auto;
+  color: black;
```

Comment:
Color `black` should be written in hexadecimal form as `#000000`<br>Color literals like `black` should only be used in variable declarations; they should be referred to via variable everywhere else.